saleor/discount/migrations/0023_voucherchannellisting.py

```diff
@@ -0,0 +1,90 @@
+# Generated by Django 3.1 on 2020-09-21 08:26
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def migrate_voucher_data(apps, schema_editor):
+    Channel = apps.get_model("channel", "Channel")
+    Voucher = apps.get_model("discount", "Voucher")
+    VoucherChannelListing = apps.get_model("discount", "VoucherChannelListing")
+
+    if Voucher.objects.exists():
+        channels_dict = {}
+
+        for voucher in Voucher.objects.iterator():
+            currency = voucher.currency
+            channel = channels_dict.get(currency)
+            if not channel:
+                channel, _ = Channel.objects.get_or_create(
+                    currency_code=currency,
+                    defaults={
+                        "name": f"Channel {currency}",
+                        "slug": f"channel-{currency.lower()}",
```

Comment:
Minor thing but we could just use `slugify` from Django. Also in recent PR of @d-wysocki we were getting or creating the default channel in a slightly different way, not sure if this is a problem or not but ideally it should be the same in all migrations: https://github.com/mirumee/saleor/pull/6137/files#diff-a5ebba8a0309c70de0585dd8c9e81b17R15-R17

Reply:
I'm add it to the feature branch to-do list. We need to fix it in all mulitchanel migrations.